packages/now-next/src/index.ts

```diff
@@ -209,7 +210,7 @@ export const build = async ({
       const { forked, getUrl } = startDevServer(entryPath, runtimeEnv);
       urls[entrypoint] = await getUrl();
       childProcess = forked;
-      console.log(
+      debug(
```

Comment:
@sophearak Hi, I just noticed that there's no dev server information when running in newer now-cli. Since this information will show only once, wouldn't it be better if we revert `debug` to `console.log`?

Reply:
👋 @judewang The reason behind this we want to have cleaner output by default.

Assuming you're using Now CLI, If you want to more verbose information, you can use `--debug` flag.

Hope that help 🙌

Reply:
@sophearak Sure I have read the discussion before. Without `--debug`, we can no longer knowing that there's a dev server running on PORT `4000` or `5000` or wherever random port, and this change wasn't be point out in the README and `now dev` official documentation. If we don't know it, maybe we would never access the dev server, but we need the dev server indeed. Just think about isn't this `Running dev server in PORT 5000` a necessary information like the `Ready! Available at http://localhost:3000` ?